packages/tools/e2e-tests/playwright.config.ts

```diff
@@ -0,0 +1,36 @@
+import { defineConfig } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: 'src/tests/*',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 1 : 0,
+  workers: process.env.CI ? 4 : 1,
+  reporter: [['github'], ['list'], ['html', { open: 'never' }]],
+  use: {
+    headless: !!process.env.CI,
+    baseURL: process.env.PLAYWRIGHT_BASE_URL
+      ? process.env.PLAYWRIGHT_BASE_URL
+      : 'http://127.0.0.1:3000',
+    channel: 'chromium',
+    trace: `retain-on-failure`,
+  },
+  expect: {
+    timeout: 10 * 1000,
+  },
+  webServer: {
+    command: `pnpm --filter ${process.env.TESTOBJECT} run start`,
+    url: 'http://127.0.0.1:3000',
+    reuseExistingServer: !process.env.CI,
```

Comment:
What happens when this is an empty string? For clarity could be `process.env.CI === undefined`

Reply:
Empty string assume false, but i'll change it either way to make it more clear.